src/main/java/org/tron/common/storage/DepositImpl.java

```diff
@@ -361,16 +355,15 @@ public synchronized long addBalance(byte[] address, long value) {
       return balance;
     }
 
-    if (value < 0 && balance < -value) {
-      throw new RuntimeException(
-          StringUtil.createReadableString(accountCapsule.createDbKey())
-              + " insufficient balance");
-    }
+//    if (value < 0 && balance < -value) {
+//      throw new ContractExeException(Hex.toHexString(accountCapsule.createDbKey())
+//              + " insufficient balance");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbKHCxjO08i52vdhEhO&open=AWbKHCxjO08i52vdhEhO)

[ISSUE_KEY]: AWbKHCxjO08i52vdhEhO